cernopendata/modules/theme/assets/semantic-ui/js/search/app.js

```diff
@@ -31,7 +31,6 @@ const initSearchApp = createSearchAppInit({
   "LayoutSwitcher.element": CODLayoutSwitcher,
   "ResultsGrid.container": null,
   "Count.element": ResultsCount,
-  "SearchApp.searchbarContainer": () => null,
```

Comment:
This seems to fix the problem indeed.

Two comments:

- Functionally, when I go to the home page do an empty search, I get _"No results found!"_, while on opendata-dev.cern.ch (i.e. in `master` branch) I get well all the hits.

- Cosmetically, it is OK to move to native RSK bar if it improves functionality, but can we amend the presentation a bit?  E.g. put the bar in banner-like position, or keep it where it is but move it more to the left to be more aligned with where facets start?

Reply:
- I checked the functionality again, I am getting all the hits locally. Did webpack recompiled again? Can you please check once again after recompiling the JS.

- The search bar is highly nested, I am currently checking if I can override some css to change the alignment.

Reply:
> I checked the functionality again, I am getting all the hits locally. Did webpack recompiled again?

I confirm that it works :+1: Perhaps I must have cleaned the instance yesterday without loading records anew?!

```console
$ docker-compose down -v
$ docker-compose build
$ docker-compose up -d
$ docker exec -i -t opendatacernch_web_1 scripts/populate-instance.sh --skip-records
$ docker exec -i -t opendatacernch_web_1 cernopendata fixtures records --mode insert-or-replace -f cernopendata/modules/fixtures/data/records/cms-primary-datasets.json
$ docker exec opendatacernch_nginx_1 find /var/cache/nginx -type f -delete
$ firefox http://localhost/search
```

Reply:
> The search bar is highly nested, I am currently checking if I can override some css to change the alignment.

Another small cosmetic thing you can fix: (a) on the home page, the search box has "Start typing..."; (b) on the search results page, the search box has "Type something".  We should unify the text to say the same thing.  Perhaps "Start typing..." bu I would simply erase the hint and not display anything.  Because each user knows how to search... We don't need to hint at anything! The search boxes can remain clean and zen.